app/controllers/discussions_controller.rb

```diff
@@ -10,11 +10,13 @@ def index
     if params[:fid]
       @discussions = Discussion.about_fid params[:fid]
     else
+      @recently_commented = Discussion.recently_commented.all
```

Comment:
in recent view there are 2 elements used to render discussions:

```
Discussion.recently_created
@recently_commented
```

I would advice to move also `Discussion.recently_created` into controller.